src/expressCassandra.js

```diff
@@ -126,6 +126,11 @@ CassandraClient.maxTimeuuid = (date) => (cql.types.TimeUuid.max(date));
 CassandraClient.minTimeuuid = (date) => (cql.types.TimeUuid.min(date));
 
 CassandraClient.prototype.doBatch = function f(queries, options, callback) {
+  if (arguments.length === 2) {
+    callback = options;
+    options = {};
+  }
+
```

Comment:
Looks like it's missing the default prepare: true part. As per doc, all queries are prepared by default. The following should be added here:

```js
const defaults = {
    prepare: true,
  };

  options = _.defaultsDeep(options, defaults);
```